lighthouse-core/config/perf.json

```diff
@@ -2,8 +2,23 @@
   "passes": [{
     "recordNetwork": true,
     "recordTrace": true,
-    "gatherers": []
-  }],
+    "gatherers": [
+      "url",
+      "image-usage",
```

Comment:
doesn't look like it will matter, but any reason to include these in the first pass instead of the second?